website/sidebars.json

```diff
@@ -1,11 +1,11 @@
 {
   "docs": {
-    "Getting started": ["en/readme"],
     "Contributing": [
       "en/contributing/install",
       "en/contributing/working",
       "en/contributing/standards",
       "en/contributing/testing",
+      "en/contributing/creating-packages",
```

Comment:
The content looks great.  I just might name the page/url "en/contributing/packages".  since the content includes creating, using, among other things.  Also, then it's more consistent until we get enough content when we need to break out for creating/editing/deleting etc.

Reply:
Is package creation under Contributors?  Will Contributors be creating new packages?  Is it more likely collaborators would create packages?

Reply:
I read this as part of general "contributing" as an act - since collaborators are contributors/contributing I think this is correct. Everything that currently exists in this section (contributing) would apply to collaborators as well, I believe (?).

Reply:
Yeah, I think this is about the act of contributing, not the role of contributor.
Name changed.